Error! This experimental syntax requires enabling one of the following parser plugin(s): 'flow, typescript' (6:7)   /home/circleci/project/node_modules/@octokit/openapi-types/generated/types.ts

## Description

I've started getting a failing pipeline (something changed upstream it seems) on a project that uses `@oktokit/rest` package. It started failing about 2 weeks ago.

The error references this package `Error! This experimental syntax requires enabling one of the following parser plugin(s): 'flow, typescript' (6:7) /home/circleci/project/node_modules/@octokit/openapi-types/generated/types.ts`
I have tried downgrading the version of `@oktokit/rest` but still seeing the issue, perhaps I can't find the right version at which the package was introduced and many dependencies are not pinned so it is tricky to find. 

The package lockfile is *not* published with you releases so we end up getting whatever is the latest version within the range and I think this is the issue really, would someone be able to confirm? 

```
snyk-api-import@ snyk-api-import
├─┬ @octokit/rest@18.0.6 invalid
│ ├─┬ @octokit/core@3.2.2
│ │ ├─┬ @octokit/auth-token@2.4.4
│ │ │ └── @octokit/types@6.0.1 deduped
│ │ ├─┬ @octokit/graphql@4.5.8
│ │ │ ├── @octokit/request@5.4.11 deduped
│ │ │ ├── @octokit/types@6.0.1 deduped
│ │ │ └── universal-user-agent@6.0.0
│ │ ├─┬ @octokit/request@5.4.11
│ │ │ ├─┬ @octokit/endpoint@6.0.10
│ │ │ │ ├── @octokit/types@6.0.1 deduped
│ │ │ │ ├── is-plain-object@5.0.0 deduped
│ │ │ │ └── universal-user-agent@6.0.0
│ │ │ ├─┬ @octokit/request-error@2.0.4
│ │ │ │ ├── @octokit/types@6.0.1 deduped
│ │ │ │ ├── deprecation@2.3.1 deduped
│ │ │ │ └── once@1.4.0 deduped
│ │ │ ├── @octokit/types@6.0.1 deduped
│ │ │ ├── deprecation@2.3.1
│ │ │ ├── is-plain-object@5.0.0
│ │ │ ├── node-fetch@2.6.1
│ │ │ ├─┬ once@1.4.0
│ │ │ │ └── wrappy@1.0.2
│ │ │ └── universal-user-agent@6.0.0
│ │ ├─┬ @octokit/types@6.0.1
│ │ │ ├── @octokit/openapi-types@1.2.2
│ │ │ └── @types/node@12.19.8 deduped
│ │ ├── before-after-hook@2.1.0
│ │ └── universal-user-agent@6.0.0
│ ├── @octokit/plugin-paginate-rest@2.6.1 extraneous
│ ├── @octokit/plugin-request-log@1.0.2
│ └── @octokit/plugin-rest-endpoint-methods@4.2.0 extraneous
├─┬ @semantic-release/exec@5.0.0
```

## More info
- Repo: https://github.com/snyk-tech-services/snyk-api-import
- Command: `npx pkg . -t node12-linux-x64,node12-macos-x64,node12-win-x64`
- This package did not used to be there in the tree (locally I had a lockfile which did not have it and the `pkg` command succeeded, unfortunately I deleted it to make sure CI and local were in sync it so can't check the versions of the dependencies that last worked :(. Realised too late that I really needed that! And it was ignored in git 🤦  so no backups for me)

## Full error
```
Error! This experimental syntax requires enabling one of the following parser plugin(s): 'flow, typescript' (6:7)
  /home/circleci/project/node_modules/@octokit/openapi-types/generated/types.ts
    at makeError (/home/circleci/project/node_modules/@semantic-release/exec/node_modules/execa/lib/error.js:59:11)
    at handlePromise (/home/circleci/project/node_modules/@semantic-release/exec/node_modules/execa/index.js:114:26)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async module.exports (/home/circleci/project/node_modules/@semantic-release/exec/lib/exec.js:16:11)
    at async prepare (/home/circleci/project/node_modules/@semantic-release/exec/index.js:54:5)
    at async validator (/home/circleci/project/node_modules/semantic-release/lib/plugins/normalize.js:34:24)
    at async /home/circleci/project/node_modules/semantic-release/lib/plugins/pipeline.js:37:34
    at async Promise.all (index 0)
    at async next (/home/circleci/project/node_modules/p-reduce/index.js:16:18) {
```

This exact same issue (pkg + octokit) has also just started occurring to us, breaking our pipeline.
Since there are no new releases of pkg since Jun (4.4.9), I thought the problem would be related to this package. However, it seems to affect others not using octokit:
https://github.com/vercel/pkg/issues/908
https://github.com/vercel/pkg/issues/672
Unfortunately the solutions described there seem to be removing/updating the offending dependencies. 

I have tried downgrading Octokit to 16.43.2 (which I knew worked with pkg@4.4.9 when building our previous release binary) but the issue persists :(

Deleting the file (node_modules/@octokit/openapi-types/generated/types.ts) right before packaging seems to solve it, but it's a bit hacky. This generates a warning when creating the package ("Warning Cannot find module '@octokit/openapi-types' from '/src'"), but everything seems to work normally with the binary.

--- a/generated/types.js
+++ b/generated/types.js
@@ -1,0 +1,1 @@
+// https://github.com/octokit/openapi-types.ts/issues/16#issuecomment-772784156

--- a/package.json
+++ b/package.json
@@ -5,7 +5,7 @@
   },
   "version": "1.0.0",
   "description": "Generated TypeScript definitions based on GitHub's OpenAPI spec",
-  "main": "generated/types.ts",
+  "main": "generated/types.js",
   "types": "generated/types.ts",
   "repository": "https://github.com/octokit/openapi-types.ts",
   "keywords": [],

